reference/json/BigCommerce_Process_Payment_API.oas2.json

```diff
@@ -652,8 +652,27 @@
         "title",
         "type"
       ]
-    }
-  },
+    },
+  "StoredPayPalAccount": {
+    "title": "Stored Paypal Account",
+    "type": "object",
+    "properties": {
+      "type": {
+        "description": "Type to classify this payment instrument"
+      },
+      "token": {
+      "description": "Identifier representing this stored paypal account",
+      "type": "string",
+      "minLength": 64,
+      "maxLength": 64
+        }
+      },
+    "required": [
+    "type",
+    "token"
+    ]
+  }
```

Comment:
Probably also need to add `PaymentInstrument` object to definitions and include `StorePayPalAccount` reference: 

```yaml
    PaymentInstrument:
      oneOf:
        - $ref: "#/components/schemas/Card"
        - $ref: "#/components/schemas/StoredCard"
        - $ref: "#/components/schemas/StoredPayPalAccount"
      discriminator:
          propertyName: type
          mapping:
            card: "#/components/schemas/Card"
            stored_card: "#/components/schemas/StoredCard"
            stored_paypal_account: "#/components/schemas/StoredPayPalAccount"
```